images/mythtv-backend/README.md

```diff
@@ -1,20 +1,39 @@
 ## mythtv-backend
 [![](https://images.microbadger.com/badges/version/instantlinux/mythtv-backend.svg)](https://microbadger.com/images/instantlinux/mythtv-backend "Version badge") [![](https://images.microbadger.com/badges/image/instantlinux/mythtv-backend.svg)](https://microbadger.com/images/instantlinux/mythtv-backend "Image badge") [![](https://images.microbadger.com/badges/commit/instantlinux/mythtv-backend.svg)](https://microbadger.com/images/instantlinux/mythtv-backend "Commit badge")
 
-The MythTV backend built under Ubuntu 16.04 LTS.
+The MythTV backend built under Ubuntu 18.04 LTS.
 
 ### Usage
 
 This image must be run in network_mode:host in order to communicate with HD Homerun tuners; assign a new IP address and hostname for this application, and define it as a secondary IP address on your Docker host's primary interface.
 
 For configuration, see the example docker-compose.yml. Set environment variables and secrets as defined here, then run "docker-compose up -d".
 
+As an alterantive, this can be run directly using environment variables. The mythtv user password cannot be setup this way, but you shouldn't be running ssh after initial setup anyway.
+
+Use -v options to map in the paths to your media. For my purposes, I've mapped a single folder into /dvr.
+~~~
+docker run -d --name mythtv \
+--network=host \
+-e DBNAME='mythtv' \
+-e DBSERVER='<Your mysql server name or ip>' \
+-e DBPASSWORD='<Password for Mythtv User>' \
```

Comment:
Passing credentials via environment variables is generally less secure than passing them by reference to a (protected from read except by root) file, which is why I set up the docker-compose file the way I do.  (A couple of links: [postgres #111](https://github.com/docker-library/postgres/issues/111), [Baslio Vera's blog](https://medium.com/@basi/docker-environment-variables-expanded-from-secrets-8fa70617b3bc)). I like the rest of your changes so I'll be happy to merge if you take out the password variable reference.

In the docker run command example you've provided here as an alternative to docker-compose, you can pass a file containing the secret:
```
docker run...
  -v /var/adm/secrets/mythtv-db-password:/run/secrets/mythtv-db-password:ro
```